website/src/data/events-listing.json

```diff
@@ -3,15 +3,9 @@
   "Livestream": [
     {
       "eventTitle": "Sourcegraph Online Meetup",
```

Comment:
I think we'd gather more interest if the title reflected the topic of the meetup, e.g. Our journey to all remote.

Reply:
yes, will do for the upcoming events.